File: Lógica/Lista2/ExPySalaC/ex4.c
#include <stdio.h>

void main()
{   
    float ladoA,ladoB,ladoC;
    
    printf("Digite o valor do lado A: ");
    scanf("%f",&ladoA);
    
    printf("Digite o valor do lado B: ");
    scanf("%f",&ladoB);
    
    printf("Digite o valor do lado C: ");
    scanf("%f",&ladoC);
    
    if (ladoA + ladoB > ladoC && ladoB + ladoC > ladoA && ladoC + ladoA > ladoB){
        if (ladoB == ladoA && ladoA == ladoC && ladoC == ladoB ){
            printf("Equilátero");
        }
        else if (ladoB == ladoA && ladoA != ladoC || ladoC == ladoA && ladoA != ladoB || ladoB == ladoC && ladoC != ladoA){
            printf("Isósceles");
        }
        else{
            printf("Escaleno");
        }
    }
    else{
        printf("Não é um triângulo");
    }
}

File: Lógica/Lista3/ExSala/Python/ex4.py
ladoA = float(input("Digite o valor do lado A: "))
ladoB = float(input("Digite o valor do lado B: "))
ladoC = float(input("Digite o valor do lado C: "))

if (ladoA + ladoB > ladoC) and (ladoB + ladoC > ladoA) and (ladoC + ladoA > ladoB):
    if (ladoB == ladoA and ladoA == ladoC and ladoC == ladoB ):
        print("Equilatéro")
    elif (ladoB == ladoA and ladoA != ladoC) or (ladoC == ladoA and ladoA != ladoB) or (ladoB == ladoC and ladoC != ladoA):
        print("Isósceles")
    else:
        print("Escaleno")
else:
    print("Não é um triângulo!!!!!!!!!")
